core/storage/exploration/gae_models.py

```diff
@@ -96,6 +96,7 @@ class ExplorationModel(base_models.VersionedModel):
     default_skin = ndb.StringProperty(default='conversation_v1')
     # DEPRECATED in v2.5.4. Do not use.
     skin_customizations = ndb.JsonProperty(indexed=False)
+    image_id_counter = ndb.JsonProperty(default=feconf.DEFAULT_IMAGE_ID_COUNTER, indexed=False)
```

Comment:
num_image_ids (and num_content_ids, similarly)

Reply:
@seanlip, Thanks for the suggestion

@import-keshav, is the going to be an Integer property or a JSON property?   (I think it's an integer, right?)

Reply:
> is the going to be an Integer property or a JSON property?

it should be an integer property.

So what will be the finalized image_id_counter name
1) image_counter
2) num_image_ids
@DubeySandeep @nithusha21 @seanlip

Reply:
Ping @seanlip

Reply:
@import-keshav, which do you think is clearer? I still like num_image_ids because that makes clear that it's a number, but would like to hear your thoughts too. I feel like image_counter represents an object of some kind.